### PR TITLE
config.yaml: enable setting of repo priority in version_os_repo_mapping

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://github.com/SUSE/sesdev/issues/162
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: seslib/templates/provision.sh.j2 [hunk 1]
```diff
@@ -29,9 +29,14 @@ sed -i 's/^rpm\.install\.excludedocs.*$/# rpm.install.excludedocs = no/' /etc/zy
 zypper ar {{ os_repo_url }} {{ os_repo_name }}
 {% endfor %}
 
-{% for _repo in version_repos %}
-zypper ar {{ _repo }} {{ version }}-repo{{ loop.index }}
-zypper mr -p 98 {{ version }}-repo{{ loop.index }}
+{% for _repo in version_repos_prio %}
+zypper ar
+{%- if _repo.priority -%}
```
Comment (toabctl) [inline seslib/templates/provision.sh.j2]: Something wrong here: `jinja2.exceptions.TemplateSyntaxError: unexpected 'end of statement block'
`. See https://ceph-ci.suse.de/blue/organizations/jenkins/sesdev-integration/detail/PR-163/2/pipeline#step-88-log-68

**Comment by smithfarm** [comment]
Setting DNM while I work through the issues

File: seslib/templates/provision.sh.j2 (hunk 1 above)
Comment (smithfarm) [reply seslib/templates/provision.sh.j2]: fixed

**Comment by smithfarm** [comment]
This has passed tests (mine and Jenkins's).

**Comment by smithfarm** [comment]
This is ready for review.

**Pull request merged**